vector/src/main/res/values/strings.xml

```diff
@@ -3222,6 +3222,7 @@
     <string name="device_manager_other_sessions_view_all">View All (%1$d)</string>
     <string name="device_manager_other_sessions_description_verified">Verified · Last activity %1$s</string>
     <string name="device_manager_other_sessions_description_unverified">Unverified · Last activity %1$s</string>
+    <string name="device_manager_other_sessions_description_inactive">Inactive for %1$d+ days (%2$s)</string>
```

Comment:
ElementBot is right (I did not think about it) please use a `plurals`. In English it will not change anything (since I do not think we will use the value `1`, but on some other languages we can have difference.

Reply:
Fixed in [5791a4d](https://github.com/vector-im/element-android/pull/6965/commits/5791a4d3cdcc7ca096f5ccdc4e2d650027919c31)